src/utils.js

```diff
@@ -66,6 +71,12 @@ const unixSeconds = () => (Math.round(Date.now() / 1000));
  */
 const secondsToPrettyTime = seconds => {
 	let timeObj = _secondsToTimeObj(seconds);
+
+	if (timeObj.days) {
+		return timeObj.days + ':' + timeObj.hours + ':' +
+			_paddedValue(timeObj.minutes) + ':' + _paddedValue(timeObj.seconds);
```

Comment:
Currently this is returning a non padded value for the hours in case hours are less than 10. So for example, if I start the timer with seconds set to `451810` it returns `5:5:30:10` instead of `5:05:30:10`. Try returning the timeObj.hours by passing it to `_paddedValue` the way we are doing for minutes and seconds.